SPECS/rpm/rpm.spec

```diff
@@ -75,6 +75,7 @@ Requires:       %{name}-libs = %{version}-%{release}
 Summary:        Binaries, scripts and libraries needed to build rpms.
 Requires:       %{name}-build-libs = %{version}-%{release}
 Requires:       %{name}-devel = %{version}-%{release}
+Requires:       azurelinux-rpm-macros >= 2.0-22
```

Comment:
Since the first version of azurelinux-rpm-macros is 2.0-26 (created in this PR), should that be the minimum version?

Reply:
Good idea. It's going to be less confusing.